zipkin/src/main/java/com/linecorp/armeria/common/tracing/RequestContextCurrentTraceContext.java

```diff
@@ -149,9 +180,32 @@ static void loadMe() {
 
             static {
                 logger.warn("Attempted to propagate trace context, but no request context available. " +
-                            "Did you remember to use RequestContext.contextAwareExecutor() or " +
-                            "RequestContext.makeContextAware()");
+                            "Did you forget to use RequestContext.contextAwareExecutor() or " +
+                            "RequestContext.makeContextAware()?");
             }
         }
     }
+
+    /** Hack to allow us to peek inside a current trace context implementation. */
+    static final class PingPongExtra {
```

Comment:
`@VisibleForTesting`, because it would have been `private` without the test cases.